client/src/components/Feed/PostSocial.js

```diff
@@ -147,16 +147,17 @@ const PostSocial = ({
       <span></span>
 
       {postId ? (
-        <div id={gtmTag("share")} className="social-icon">
+        <div id={gtmTag("share", GTM.post.prefix)} className="social-icon">
           <CopyToClipboard text={url} onCopy={onCopyLink}>
             <span>
               {renderShareIcon()}
-              <span className="social-text">Share</span>
+              <span className="total-number">{numShares}</span>
```

Comment:
Did we decide to display the number of shares? (Also, don't know if it is actually working, please resolve merge conflicts first.

Reply:
Yeah this one is not ready @jyclam, can we work on this one now? I will resolve, remove extra commit and push in a few mins but afterwards we should work on the other stuff that is missing

Reply:
GT has head out for the night, and I will head out soon :(

Reply:
This branch doesn't seem to be complete and it has merge conflicts.

Reply:
@idjevm Sure, I can take over but I likely won't finish it until tomorrow.